### DISCUSSION TITLE
Semver

### DISCUSSION THREAD

**Discussion by jasongitmail** [etherCorps/sveltekit-og#12] [General]
I noticed breaking changes are currently using a patch x.x.**x** version update. 

I wanted to encourage the use of [semver](https://docs.npmjs.com/cli/v6/using-npm/semver) for versioning so that NPM updates will work predictably and related tooling will work predictably that way, like https://www.npmjs.com/package/npm-check

**Comment by theetherGit** [comment]
I'll keep that in mind for future reference.
